File: dotnet/DesktopAutomation-Series/WinAppDriverPageObjects/Views/StandardView.Assertions.cs
using NUnit.Framework;

namespace WinAppDriverPageObjects.Views
{
    public partial class CalculatorStandardView
    {
        public void AssertResult(decimal expectedReslt)
        {
            string strResult = GetCalculatorResultText();
            var actualResult = decimal.Parse(strResult);

            Assert.AreEqual(expectedReslt, actualResult);
        }
    }
}

File: vbnet/DesktopAutomation-Series/WinAppDriverPageObjects/Views/StandardView.Assertions.vb
Imports NUnit.Framework

Namespace WinAppDriverPageObjects.Views
    Public Partial Class CalculatorStandardView
        Public Sub AssertResult(ByVal expectedReslt As Decimal)
            Dim strResult As String = GetCalculatorResultText()
            Dim actualResult = Decimal.Parse(strResult)
            Assert.AreEqual(expectedReslt, actualResult)
        End Sub
    End Class
End Namespace
